components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -1778,8 +1798,14 @@ public API updateAPI(API api, API existingAPI) throws APIManagementException, Fa
                 }
             }
             try {
+                Organization org;
+                if (api.getOrganizationId() != null) {
+                    org = new Organization(api.getOrganizationId());
+                } else {
+                    org = new Organization(tenantDomain);
+                }
```

Comment:
could avoid the If block in the same way and make it a single line?

Reply:
We need to set the org Object with either the organizationId from API or with the tenant domain. In a single line, we can't achieve this nah ?